### PR TITLE
feat: updated seeder projects to .net8

### PR DESCRIPTION
**Pull request by navodwickra1**
| Action | Done? |
| --- | --- |
| Code compiles and unit tests all pass locally | ✔ |
| Debug/console log code removed | ❌ |
| Commented out code removed | ✔ |
| Unit tests added/updated | ❌ |
| README updated | ❌ |
| Considered: <br /> - Performance <br /> - Security <br /> - Logging | ❌ |
| Licenses of any new/upgraded dependencies checked, with no copyleft dependencies introduced | ✔ |

### CODE REVIEW

File: openiddict/src/Audacia.Auth.OpenIddict.Seeding.EntityFramework/EntityFrameworkSeeder.cs [hunk 1]
```diff
@@ -29,7 +29,10 @@ public EntityFrameworkSeeder(string identityProjectBasePath, string identityProj
     protected override IServiceCollection ConfigureAdditionalServices(IServiceCollection services, IConfiguration configuration)
     {
         var connectionString = configuration.GetConnectionString(_connectionStringName);
-        services.AddScoped(_ => new OpenIddictContext<TKey>(connectionString));
+        if (connectionString != null)
```
Comment (Audacia-RhysSmith) [inline openiddict/src/Audacia.Auth.OpenIddict.Seeding.EntityFramework/EntityFrameworkSeeder.cs:32]: If we are checking if the connection string is not null, should we continue if it is null?

Comment (navodwickra1) [reply openiddict/src/Audacia.Auth.OpenIddict.Seeding.EntityFramework/EntityFrameworkSeeder.cs:32]: have changed it to throw ConfigurationErrorsException instead

**Review by jackpercy-acl — COMMENTED** [review]
@navodwickra1 appreciate this is a blocker for you, but there had been some discussions about v4 of Audacia.Auth supporting multi-targeting as this is what OpenIddict does.

Slightly concerned if we bump Audacia.Auth up to v4 with only .NET 8 Support, it would be odd for us to then re-introduce .NET 6 support with multi-targeting in v5.

Do you think the multi-targeting approach would be something you have time to look into?

I think the only sort of gotcha with this is that other dependencies will need different versions based on the targeted framework, e.g. EF Core and ASP.NET will need v6/v8 depending on the target.

You can see how this has been implemented in OpenIddict here, so might be something we can use as inspiration - https://github.com/openiddict/openiddict-core/blob/dev/Directory.Packages.props

**Comment by jackpercy-acl** [comment]
> @navodwickra1 appreciate this is a blocker for you, but there had been some discussions about v4 of Audacia.Auth supporting multi-targeting as this is what OpenIddict does.
> 
> Slightly concerned if we bump Audacia.Auth up to v4 with only .NET 8 Support, it would be odd for us to then re-introduce .NET 6 support with multi-targeting in v5.
> 
> Do you think the multi-targeting approach would be something you have time to look into?
> 
> I think the only sort of gotcha with this is that other dependencies will need different versions based on the targeted framework, e.g. EF Core and ASP.NET will need v6/v8 depending on the target.
> 
> You can see how this has been implemented in OpenIddict here, so might be something we can use as inspiration - https://github.com/openiddict/openiddict-core/blob/dev/Directory.Packages.props

@navodwickra1 the multi-targeting PR (https://github.com/audaciaconsulting/Audacia.Auth/pull/5) has been completed - thanks for your help with testing this on .NET 8.

That supersedes this PR so please can you close off this one.

**Pull request closed without merging**